### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by mistermaxis**
### Tasks Completed:
- Add species column.
- Add additional animal data.
- Add queries.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -15,3 +15,34 @@ select * from animals where neutered is true;
 select * from animals where name != 'Gabumon';
 
 select * from animals where weight_kg between 10.4 and 17.3;
+
+begin;
+update animals set species = 'Digimon' where name like '%mon';
+update animals set species = 'Pokemon' where name not like '%mon';
```
Comment (descholar-ceo) [inline queries.sql:21]: - [ ] You should set species = pokemon if name is null

File: queries.sql [hunk 2]
```diff
@@ -15,3 +15,34 @@ select * from animals where neutered is true;
 select * from animals where name != 'Gabumon';
 
 select * from animals where weight_kg between 10.4 and 17.3;
+
+begin;
+update animals set species = 'Digimon' where name like '%mon';
```
Comment (descholar-ceo) [inline queries.sql:20]: - [ ] We are missing one more query which updates species and set them to unspecified

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @mistermaxis , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @mistermaxis 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**